.github/workflows/ci.yml

```diff
@@ -183,6 +189,9 @@ jobs:
           docker exec --user root unreal bash -c "
             chown -R $uid /home/ue4/UnrealEngine/Engine/Binaries/ThirdParty/Mono/Linux ;
             chown -R $uid /home/ue4/UnrealEngine/Engine/Binaries/ThirdParty/DotNet "
+          # Remove corrupted engine files that have to be rebuild
```

Comment:
Why are there corrupted engine files?

Reply:
It's an engine bug yet I'll remove this for now since the fix is all about network setup.